query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/notUsingSchemaBase/NestedUpdateMutationInsideUpdateSpec.scala

```diff
@@ -7,57 +7,6 @@ import util._
 class NestedUpdateMutationInsideUpdateSpec extends FlatSpec with Matchers with ApiSpecBase with SchemaBaseV11 {
   override def runOnlyForCapabilities: Set[ConnectorCapability] = Set(JoinRelationLinksCapability)
 
-  "A P1! to C1! relation relation" should "work" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.